internal/block/block_test.go

```diff
@@ -457,3 +457,63 @@ func TestMarshal(t *testing.T) {
 		})
 	}
 }
+
+func TestUmarshal(t *testing.T) {
+	testJSONBlock := JSONBlock{
```

Comment:
Can you call `block.Marshall` on a regular block and use that output as the `JSONBlock`? I just want to make sure it's consistent with that function.